src/App.js

```diff
@@ -1,54 +1,83 @@
 import { useEffect, useState } from "react";
-import './App.css';
-import logo from './mlh-prep.png'
+import "./App.css";
+import logo from "./mlh-prep.png";
+import WeatherMap from "./components/WeatherMap";
 
 function App() {
   const [error, setError] = useState(null);
   const [isLoaded, setIsLoaded] = useState(false);
-  const [city, setCity] = useState("New York City")
+  const [city, setCity] = useState("New York City");
   const [results, setResults] = useState(null);
+  const [cityCoordinates, setCityCoordinates] = useState({
+    lat: "51.505",
+    lon: "-0.09",
+  });
 
   useEffect(() => {
-    fetch(`https://api.openweathermap.org/data/2.5/weather?q=${city}&units=metric&appid=${process.env.REACT_APP_APIKEY}`)
-      .then(res => res.json())
+    fetch(
+      `https://api.openweathermap.org/data/2.5/weather?q=${city}&units=metric&appid=${process.env.REACT_APP_APIKEY}`
+    )
+      .then((res) => res.json())
       .then(
         (result) => {
-          if (result['cod'] !== 200) {
-            setIsLoaded(false)
+          if (result["cod"] !== 200) {
+            setIsLoaded(false);
           } else {
             setIsLoaded(true);
             setResults(result);
+            setCity(result.name);
+            setCityCoordinates({
+              lat: result.coord.lat,
+              lon: result.coord.lon,
+            });
           }
         },
         (error) => {
           setIsLoaded(true);
           setError(error);
         }
-      )
-  }, [city])
+      );
+  }, [city]);
```

Comment:
```suggestion
      )
  }, [city])
```

this is correct but lets revert for now